src/config.rs

```diff
@@ -32,11 +32,18 @@ impl Default for ImageConfig {
         let work_dir = PathBuf::from(
             std::env::var(CC_IMAGE_WORK_DIR).unwrap_or_else(|_| DEFAULT_WORK_DIR.to_string()),
         );
-
         ImageConfig {
             work_dir,
             default_snapshot: SnapshotType::Overlay,
-            security_validate: false,
+            security_validate: std::env::var(ENABLE_SECURITY_VALIDATE).map_or_else(
```

Comment:
security validate is a key config to enable signature verification, first should we enable it by default? second, environment variable is not measured, the host may inject the envrionment variable to guest to disable our verification @sameo @jialez0 @Xynnn007 any suggestions?

Reply:
https://github.com/kata-containers/kata-containers/issues/4888

Reply:
currently image-rs itself does not allow control variables to be passed down during pull_image(), and the ocicrypt-rs control reads its config separately inside image-rs instance, so my quick suggestion is as bellow:

1. with confidential containers cases, default to enable image verification and we have a switch in agent config(both Kata agent and enclave-cc agent) to provide a second choice for user with their serious consideration to disable it
2. image-rs to allow configuration to be sent down during pull_image() to allow caller to tune the exact behavior of image-rs in run time

Reply:
Hi @hairongchen @HaokunX-intel , with @stevenhorsman 's PR in `kata-agent`: https://github.com/kata-containers/kata-containers/pull/5552 , we can set `enable_signature_verification` config in kernel param of kata config file, can this meet your requirement?

Reply:
Hi @arronwy thanks for the updates, we will update this PR to follow this method, i.e.: enclave-agent will read config from agent config file and set enable_signature_verification accordingly